haystack/nodes/answer_generator/transformers.py

```diff
@@ -367,6 +368,10 @@ def __init__(
                         [torch.device('cuda:0'), "mps", "cuda:1"]). When specifying `use_gpu=False` the devices
                         parameter is not used and a single cpu device is used for inference.
         """
+        warnings.warn(
+            "Seq2SeqGenerator component is deprecated and will be removed in future versions.",
```

Comment:
Should we also suggest that they should use PromptNode by saying?

"`Seq2SeqGenerator` component is deprecated and will be removed in future versions. Use `PromptNode` instead of `Seq2SeqGenerator`."

Reply:
Don't ask just push @mayankjobanputra 😉